pages/compliance/_sub-hero.html

```diff
@@ -0,0 +1,40 @@
+<div class="row row-page-header">
+  <div class="col-xs-12">
+    <h2>Compliance in hours, not months</h2>
```

Comment:
Nice 👍 

Though perhaps it should say "days" instead of "hours", as there are typically process changes you have to implement to be compliant, which takes more than hours, right?

Reply:
Yes, I guess it's likely to take days, though our eventual target might be hours, especially if integrated in to a compliance-flavored ref arch.